bin/list

```diff
@@ -86,12 +86,13 @@ const renderDirectory = async dir => {
     const index = files.indexOf(file)
     const details = path.parse(filePath)
 
-    details.ext = details.ext.split('.')[1]
     details.relative = path.join(subPath, details.base)
 
     if (await isDir(filePath)) {
       details.base += '/'
     } else {
+      details.ext = details.ext.split('.')[1] || 'txt'
```

Comment:
For files with no extension, `details.ext.split('.')[1]` is `undefined`. `details.ext` is used [here](https://github.com/zeit/list/blob/master/views/index.hbs#L26) to add a class that determines which icon is used, and if it's `undefined`, [the class that renders the directory icon](https://github.com/zeit/list/blob/master/assets/styles.scss#L116) will apply.

Using `anyrandomstring` instead of `txt` would yield the same results, but I chose `txt` in order to be consistent with how`.txt` files are handled.